src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -154,20 +155,20 @@ export function AmountField({
           position="relative"
           fontFamily="Marche"
         >
-          {isSendingMax ? <Text fontSize={fontSize + 'px'}>~</Text> : null}
-          <Input
-            _disabled={{ bg: color('bg') }}
+          {isSendingMax ? <styled.span fontSize={fontSize + 'px'}>~</styled.span> : null}
```

Comment:
I have some work to do here with these `fontSizes`. This logic seems pretty complicated though, maybe we can just make it a set size? Do we need to do this?
![Screenshot 2023-11-14 at 12 08 28](https://github.com/leather-wallet/extension/assets/2938440/84178ea3-a384-40c7-b1ba-5e101449d0ac)

Reply:
I might be able to tackle this bc I wrote the first implementation of the scaling font sizes. @mica000 any thoughts here on changing the designs?